spec/requests/api/v1/user_request_spec.rb

```diff
@@ -112,4 +112,32 @@
     expect(User.count).to eq(0)
     expect{User.find(user.id)}.to raise_error(ActiveRecord::RecordNotFound)
   end
+
```

Comment:
Spec to check that the logged in user will only match with similar users and not every user when they look for a run buddy.